ChecksProductDetailsFunctionality_test.java

```diff
@@ -0,0 +1,49 @@
+package AutomationPracticeTestsSuite;
+
+import common.BaseTest;
+import org.junit.Test;
+import org.testng.asserts.SoftAssert;
+import pageobject.AutomationPracticeSite.AutomationPracticeMainPage;
+import pageobject.AutomationPracticeSite.ProductDetailsPage;
+
+public class ChecksProductDetailsFunctionality_test extends BaseTest {
+
+    @Test
+    public void verifyProductDetailsPageFunctionality() {
+
+        String navigateUrl = propertyHelper.readProperty("automation-practice.site.url");
+        webDriver.get(navigateUrl);
+
+        String expectedResultOfProductTitle = "Printed Chiffon Dress";
+        String expectedResultOfProductPrice = "$16.40";
+        String expectedResultOfProductQuantity = "1";
+        String expectedResultOfProductSize = "S";
+        String expectedResultOfProductColor = "Yellow";
+        String expectedResultOfProductCompositions = "Polyester";
+        String expectedResultOfProductStyles = "Girly";
+        String expectedResultOfProductProperties = "Midi Dress";
+
+        AutomationPracticeMainPage mainPage = new AutomationPracticeMainPage(webDriver);
+        ProductDetailsPage detailsPage = mainPage.moveOnProductDetailsPage();
+
+        String actualResultOfProductTitle = detailsPage.getProductTitleText();
```

Comment:
another though for potential improvement - you could introduce separate class to contain all of these props, build object from you expectations & compare it with object returned from page

Reply:
I agree with you, it can be implemented this way.